docs/how-to-add-new-reader.rst

```diff
@@ -32,8 +32,8 @@ Subclassing of Engine/Reader using AutoFilterReaderEngine
 Your ``YourEngine`` should extend :py:class:`~pyaro.timeseries.AutoFilterReaderEngine.AutoFilterEngine`
 and it must implement the following methods:
 
-- :py:meth:`~pyaro.timeseries.Engine.open`: opening a reader with the signature (self, filename, *args, **kwargs) -> YourReader``,
-  usually just like: ``return YourReader(filename, *args, **kwargs)``
+- :py:meth:`~pyaro.timeseries.AutoFilterReader.reader_class`: a class implementing `AutoFilterReader`,
```

Comment:
maybe this should be more specific:

the class implementing the `AutoFilterReader` corresponding to this `AutoFilterEnginge`, e.g. the reader_class() of `YourEngine` should return `YourReader`